cloudformation/membership-attribute-service-tables.yaml

```diff
@@ -0,0 +1,45 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Dynamo tables for Membership Attributes service
+Parameters:
+  TouchpointBackend:
+    Description: Environment name ( DEV, UAT or PROD)
+    Type: String
+    Default: DEV
+Mappings:
+  StageVariables:
+    DEV:
+          DynamoWriteCapacity: 1
+          DynamoReadCapacity: 5
+    UAT:
+          DynamoWriteCapacity: 1
+          DynamoReadCapacity: 5
+    PROD:
+      DynamoWriteCapacity: 10
+      DynamoReadCapacity: 200
+Resources:
+  SupporterAttributesFallbackTable:
+    Type: "AWS::DynamoDB::Table"
+    Properties:
+      AttributeDefinitions:
+          - AttributeName: "UserId"
+            AttributeType: "S"
+      KeySchema:
+          - AttributeName: "UserId"
+            KeyType: "HASH"
+      ProvisionedThroughput:
+        ReadCapacityUnits:
+          Fn::FindInMap: [ StageVariables, { Ref: TouchpointBackend }, DynamoReadCapacity ]
+        WriteCapacityUnits:
+          Fn::FindInMap: [ StageVariables, { Ref: TouchpointBackend }, DynamoReadCapacity ]
+      SSESpecification:
+        SSEEnabled: true
+      TableName: !Sub SupporterAttributesFallback-${TouchpointBackend}
+      TimeToLiveSpecification:
+        AttributeName: "TTLTimestamp"
+        Enabled: true
+Outputs:
+  SupporterAttributesFallbackTable:
+    Description: supporter attributes fallback table
+    Value: !GetAtt [ SupporterAttributesFallbackTable, "Arn" ]
+    Export:
+      Name: !Sub ${AWS::StackName}-SupporterAttributesFallback
```

Comment:
the output parameter is exported prefixed with the name of the stack which seems to be a common practice and it's helpful for finding where an input parameter to another stack comes from. The only tradeoff that I see is that the name of the stack is manually set when it's created but it is written in the cloudformation templates that include this param so there is some room (but not a lot) for mistakes there.

I could also not use AWS::StackName in the  export name in which case we would have to search for the cfn template that includes the name to identify where an import comes from